tests/ui/test-attrs/test-panic-abort-disabled.rs

```diff
@@ -3,6 +3,7 @@
 // compile-flags: --test -Cpanic=abort -Zpanic-abort-tests=no
 // run-flags: --test-threads=1
 
+// needs-unwind
```

Comment:
why does this need unwind? does it emit a different error on targets without unwinding?

Reply:
Because of the `#[should_panic]` test.